src/autoscaler/metricsforwarder/config/config.go

```diff
@@ -86,6 +91,15 @@ func LoadConfig(reader io.Reader) (*Config, error) {
 	dec.KnownFields(true)
 	err := dec.Decode(conf)
```

Comment:
If this now fails, we loose the error-message because of line 96. This is a bug.